docs/deployment/pull-backup.rst

```diff
@@ -0,0 +1,108 @@
+.. include:: ../global.rst.inc
+.. highlight:: none
+
+Backing up in pull mode
+=======================
+
+Assuming you have a pull backup system set up with borg, where a backup server
+pulls the data from the target via sshfs.
+
+In this mode, the backup client's file system is mounted via sshfs on the backup
+server. Pull mode is even possible if the SSH connection must be established by
+the client via a remote tunnel.
+
+There are some restrictions caused by sshfs. Unless you define UID and GID
+mappings when mounting via sshfs, owners and groups of the mounted file system
+will probably change, and you may not have access to those files if BorgBackup
+is not run with root privileges.
+
+[ADD MORE RESTRICTIONS HERE: filename encoding, xattrs, ACLs, bsdflags...]
```

Comment:
I could not find useful information about restrictions, do you have reliable info? A sentence like "There may be more restictions: ..." seems not appropriate to me.

Reply:
Depending on the filesystem used (sshfs here, but could be anything), you need to read about its limitations in its own docs, that is out of scope of borg docs.

Reply:
So, the uid/username gid/groupname mapping issue is in general worth noting.
For other issues I'ld just put some sentence like:

Please make sure the filesystem you use implements all the features you need (e.g. check if it implements xattrs and ACLs, if bsdflags work, if long non-ascii filenames work).

Reply:
OK. My thoughts were "this recipe uses sshfs explicitly, so I should note what restrictions it brings". I haven't found any sshfs-specific info about restrictions yet. I started looking into FUSE docs (libfuse), but there is also nothing mentioned. I will check the FUSE kernel module docs, too. If I cannot find any useful info, I will insert your proposed sentence.

Reply:
Did some research, sshfs uses SFTP protocol, which is in IETF draft status. The definition generally does support xattrs and ACLs, but implementations may not have support. So I'll add some explanations regarding this.